NickvisionTagger.GNOME/Helpers/Help.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace NickvisionTagger.GNOME.Helpers;
+
+/// <summary>
+/// Helper class for help docs
+/// </summary>
+public static class Help
+{
+    /// <summary>
+    /// Get URL for given help page
+    /// </summary>
+    /// <param name="pageName">Help page name</param>
+    /// <returns>URL to either yelp or web page</returns>
+    public static string GetHelpURL(string pageName)
+    {
+        if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("SNAP")))
+        {
+            return $"help:tagger/{pageName}";
+        }
+        var lang = "C";
+        if (!CultureInfo.CurrentCulture.Equals(CultureInfo.InvariantCulture) && CultureInfo.CurrentCulture.Name != "en-US")
```

Comment:
You can do `CurrentCulture != CultureInfo.InvariantCulture`.

The `==` operators are implemented to call Equals (hence why we use it for `string` and not Equals)

Reply:
With `==`/`!=` I get warning `Possible unintended reference comparison. To get a value comparison, use Equals method`.

Reply:
Ah I guess for CultureInfo the operator is not overloaded...ok :)